src/lib.rs

```diff
@@ -291,13 +291,21 @@ fn get_log_file_path(
     if log_size > max_file_size {
       match rotation_strategy {
         RotationStrategy::KeepAll => {
-          fs::rename(
-            &path,
-            dir.as_ref().join(format!(
-              "{}.log",
-              chrono::Local::now().format("app-%Y-%m-%d")
-            )),
-          )?;
+          let to = dir.as_ref().join(format!(
+            "{}.log",
+            chrono::Local::now().format("app_%Y-%m-%d_%H-%M-%S")
```

Comment:
Reading through this, we actually have a far more important issue here. When we rename a logfile as part of the rotation strategy the name of the app get's lost and every file just get's renamed to `app-<date>.log`

We should pass `app_name` into the format macro here too.

Reply:
for us this was less important because no data was lost at least :D.

We put the logs in a folder with the app name to avoid any naming issues.

However I also have no problems with keeping the app name on rotation if you prefer this way.

Reply:
Yeah I know it's minor, but when users attempt to review logs using utilities such as `Console` they couldn't find the file if it's `app-<date>.log` so I would appreciate if you could make that change too!

Reply:
9239622